strapi/strapi
# feat(main-nav): Main Nav refactoring, refactor the logo

### What does it do?

In the condensed view of the Left Menu it shows the new logo. The logo is no longer a link to the home page, but just a container with an image

### Why is it needed?

In the refactoring of the Main Nav we want a new home icon to have the task of connecting to the home page, while the logo must only show the image of the logo.
The Home icon will be implemented in another PR.

### How to test it?
Disclaimer: the new logo is used only in the condensed view at the moment because I need to implement the components used in the Logo and in the new Home icon before implementing the new MainNav wrapper

1. Open a Strapi App
2. in the left Main Nav click on the reduce button, to show the condensed view of the Main Nav
3. you can now see the new version of the logo (only the width is not correct because we need to change the width of the MainNav wrapper)
4. if you try to click on the logo nothing happens

You can find all the info and link to the figma file in the Jira ticket

### Related issue(s)/PR(s)

CS-762
